packages/renderer/src/lib/image/ImagesList.svelte

```diff
@@ -214,6 +169,50 @@ function computeInterval(): number {
   // every day
   return 60 * 60 * 24 * SECOND;
 }
+
+let selectedItemsNumber: number;
+let table: Table;
+
+const columns: Column[] = [
+  new Column('Status', 'center', '70px'),
+  new Column('Name', 'left', '3fr'),
+  new Column('Age', 'left', '1fr'),
+  new Column('Size', 'right', '1fr'),
+];
+
+const tableHelper = new (class extends TableHelper {
+  getKey(object: any): any {
+    return object.name + object.shortId + object.tag;
+  }
+
+  selectable(image: any): boolean {
+    return image.inUse;
+  }
+
+  disabledText(): string {
+    return 'Image is used by a container';
+  }
+
+  compare(column: string): ((object1: any, object2: any) => number) | undefined {
+    if (column === 'Status') {
```

Comment:
switch statement instead of what if may be better / cleaner?

Reply:
Used alternate suggestion to move comparator (and renderer) to column level.